pkg/kyverno/common/fetch.go

```diff
@@ -286,8 +286,8 @@ func GetPatchedResource(patchResourceBytes []byte) (patchedResource unstructured
 	return patchedResource, nil
 }
 
-// getKindsFromPolicy will return the kinds from policy match block
-func getKindsFromPolicy(rule v1.Rule) map[string]bool {
+// GetKindsFromRule will return the kinds from policy match block
+func GetKindsFromRule(rule v1.Rule) map[string]bool {
```

Comment:
Making this change for the below two reasons:

1. First letter capital allows this function to be imported as part of the package. 
2. There does already exist a function with the same name(`GetKindsFromPolicy`) in this package.